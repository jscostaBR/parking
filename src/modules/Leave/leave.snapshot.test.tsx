import * as React from 'react'
import { Leave } from '.'
import renderer from 'react-test-renderer'

describe('render Leave', () => {
  it('render correctly', () => {
    const tree = renderer.create(
      <Leave />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
