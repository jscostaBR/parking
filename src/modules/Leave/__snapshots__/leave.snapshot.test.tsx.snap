// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Leave render correctly 1`] = `
<div
  className="sc-bczRLJ fXEewa"
>
  <form
    onSubmit={[Function]}
  >
    <label
      className="sc-gsnTZi kQVvHB"
    >
      <span>
        Número da placa:
      </span>
      <input
        name="leave"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        onPaste={[Function]}
        placeholder="AAA-0000"
        value=""
      />
    </label>
    <button
      className="sc-dkzDqf sc-eCYdqJ excwFd bMHkOw"
      disabled={true}
      onClick={[Function]}
    >
      PAGAMENTO
    </button>
    <button
      className="sc-dkzDqf sc-jSMfEi excwFd mpEnG"
      disabled={true}
      onClick={[Function]}
    >
      SAÍDA
    </button>
    <button
      className="go-to-historic"
      disabled={true}
      onClick={[Function]}
    >
      Ver Histórico
    </button>
  </form>
</div>
`;
